v3/plugins/eclipse/che-theia/next/meta.yaml

```diff
@@ -89,6 +89,8 @@ spec:
          - exposedPort: 13132
          - exposedPort: 13133
      memoryLimit: "512M"
+     cpuLimit: "1000m"
```

Comment:
@ibuziuk looks like the problem on my side was that "1000m" got translated in "1k". It works if I set "1" instead.

Reply:
sure I can change, but as I understand for you it worked according to the https://github.com/eclipse/che/issues/18472

```
apiVersion: 1.0.0
metadata:
  name: alotofcpu
components:
  - cpuLimit: 1500m
    id: eclipse/che-theia/next
    type: cheEditor
```

Reply:
according to the docs `m` seems to be the only parameter that is supported. @ericwill any input on this? 

> Kubernetes/OpenShift-spec CPU request string (e.g. "125m"). Refer to https://kubernetes.io/docs/concepts/configuration/manage-compute-resources-container/#meaning-of-cpu for details

Reply:
@ibuziuk yes specifying 1500m worked on the devfile but I was wondering if on the meta.yaml the value is parsed/converted in another part of the code. I will try to reproduce it on my local minikube.